docs/MIGRATION.md

```diff
@@ -22,7 +22,7 @@ For example:
 
 ```js
     "importOrder": [
-        "", // This emptry group at the start will add separators for side-effect imports and node.js built-in modules
+        "", // If you want a gap at the top after top-of-file-comments, put a separator here!
```

Comment:
We might want to include `"<BUILTIN_MODULES>", ""` here too, to really replicate `importOrderSeparation`

Reply:
Done!